script/playbackstrategy/modifiers/live/restartable.js

```diff
@@ -96,23 +80,37 @@ define(
           }
         }
 
+        function getCurrentTime () {
+          return fakeTimer.currentTime;
+        }
+
+        function getSeekableRange () {
+          var windowLength = (timeData.windowEndTime - timeData.windowStartTime) / 1000;
```

Comment:
As discussed, `restartable` devices should report a reliable seekable range when it first starts playing, before it becomes inaccurate over time. This will generally be shorter than the window, as browsers often struggle seeking to the extremities of the window. 

As such, it will be more correct to the initially reported seekable range from the browser a a baseline, rather than the extremities of the window.